### PR TITLE
amv improvements

### PR DESCRIPTION
**Pull request by italoacasas**
### Changelog
- `avm install 0.0.0` ignores request if `0.0.0` is already installed
- `avm install 0.0.0` switches to version `0.0.0` after successfully installation
- `avm update` install latest stable version

### Issues
https://github.com/project-serum/anchor/issues/1663

### CODE REVIEW

File: avm/src/lib.rs [hunk 1]
```diff
@@ -70,8 +70,24 @@ pub fn use_version(version: &Version) -> Result<()> {
     Ok(())
 }
 
+/// Update to latest stable version
+pub fn update() -> Result<()> {
+    // Find last stable version
+    let version = &get_latest_version();
+
+    install_version(version)?;
+    Ok(())
```
Comment (paul-schaaf) [inline avm/src/lib.rs:79]: nit: you can do `install_version` as the last line in the function.

same with `use_version` below

Comment (italoacasas) [reply avm/src/lib.rs]: done

**Comment by paul-schaaf** [comment]
pls update the `CHANGELOG.md` too

I think this is even a breaking change. because of this:
```
avm install 0.0.0 switches to version 0.0.0 after successfully installation
```

**Comment by italoacasas** [comment]
> pls update the `CHANGELOG.md` too
> 
> I think this is even a breaking change. because of this:
> 
> ```
> avm install 0.0.0 switches to version 0.0.0 after successfully installation
> ```

Yes, I agree with you.

File: CHANGELOG.md [hunk 2]
```diff
@@ -563,4 +574,4 @@ Initial release.
 * spl: `anchor-spl` crate providing CPI clients for Anchor programs.
 * client: `anchor-client` crate providing Rust clients for Anchor programs.
 * ts: `@project-serum/anchor` package for generating TypeScript clients.
-* cli: Command line interface for managing Anchor programs.
+* cli: Command line interface for managing Anchor programs.
```
Comment (paul-schaaf) [inline CHANGELOG.md:577]: what happened here? I saw that in a different PR too 🤔

Comment (paul-schaaf) [reply CHANGELOG.md:577]: probably just a github bug?

Comment (italoacasas) [reply CHANGELOG.md:577]: Visual Studio Code doing weird stuff when I saved the document 😵‍💫

Comment (paul-schaaf) [reply CHANGELOG.md:577]: oh maybe a different os line break?

Comment (italoacasas) [reply CHANGELOG.md:577]: Maybe, not sure, honestly, but I think it is more than just that. For example, if I use VSCode, when I save, all the `*` get replaced by  `-.`

**Review by paul-schaaf — APPROVED** [review]

**Pull request merged**